gdcdictionary/schemas/pathology_detail.yaml

```diff
@@ -325,6 +343,18 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  lymph_node_dissection_method:
+    enum:
+      - Functional (Limited) Neck Dissection
+      - Modified Radical Neck Dissection
+      - Radical Neck Dissection
+
+  lymph_node_dissection_site:
+    enum:
+      - Neck, Left
+      - Neck, Right
+      - Neck, NOS
```

Comment:
no idea if we need to alpha sort this. does NOS  always come after the...non-NOSes?

Reply:
Hah, I'm not sure it just looks correct, let's leave it.